languages/elixir/exercises/concept/errors/.meta/example.ex

```diff
@@ -0,0 +1,21 @@
+defmodule RPNCalculator do
```

Comment:
Reverse Polish Notation? I hope the story will be simple enough for unmathematical folks 🤓